getting-started/con-using-external-database-operator.adoc

```diff
@@ -0,0 +1,11 @@
+[id='con-using-external-database-operator_{context}']
+= Using an external database with the Operator
+
+By default, the Custom Resource (CR) is set to deploy a local PostgreSQL database, but this behavior can be disabled in the CR. To do so, you need to set the *spec.database.enableLocalDb* field to `false` and edit the *spec.database.authSecretName* field referencing a Secret containing the external DB connection information.
```

Comment:
this is an unsupported feature in 1.1. Please comment this out until we resolve https://issues.redhat.com/browse/RHIDP-453

Reply:
@nickboldt this is supported already, and the status of https://issues.redhat.com/browse/RHIDP-453 should be updated - fyi @rm3l

Reply:
Ok, after discussion with PM (see thread in #team-rhdh), let's remove/comment this even if both the Helm Chart and the Operator provide the ability to disable the local DB and use a separate Postgres DB.
If we want to mention support for external databases, we will need to also support SQL databases other than PostgreSQL, which is not yet supported by upstream Backstage.
But I think it makes sense to keep it as it is in the [`janus-idp/operator` repo](https://github.com/janus-idp/operator/blob/main/.rhdh/docs/openshift.adoc#using-an-external-database), since that's technically possible.